build-scripts/gulp/webpack.js

```diff
@@ -67,27 +67,33 @@ const handler = (done) => (err, stats) => {
   }
 };
 
+const prodBuild = (conf) =>
+  new Promise((resolve) => {
+    const compiler = webpack(
+      conf,
+      // At end of stats printing, close compiler and resolve promise
+      doneHandler(() => compiler.close(resolve))
```

Comment:
We pass a `callback` to `webpack` so the compiler is automatically closed.

```suggestion
      doneHandler(resolve)
```